index.js

```diff
@@ -144,22 +148,23 @@ module.exports = ({utPort, registerErrors}) => class HttpPort extends utPort {
             const parseResponse = this.config.parseResponse !== false && msg.parseResponse !== false;
             const reqProps = {
                 withCredentials: msg.withCredentials || this.config.withCredentials,
-                requestTimeout: msg.requestTimeout || this.config.requestTimeout || 30000
+                requestTimeout: msg.requestTimeout || this.config.requestTimeout || 30000,
+                headers: this.config.headers,
+                followRedirect: false
             };
             if (methodName && this.openApi[methodName]) {
-                Object.assign(reqProps, this.config.raw, this.openApi[methodName](msg));
+                merge(reqProps, this.config.raw, this.openApi[methodName](msg));
```

Comment:
This can potentially overwrite properties in this.config.headers if raw or openapi have properties with the same names. As result, next request can send headers from previous. Try this in node repl:
```js
x={};require('ut-function.merge')({a:{x}},{a:{x:{y:1}}});x;
// x is now {y:1}
```

Reply:
You are right. This happens because the merge function overrides the target object by reference. 
But we discussed with Kris that one might want to set headers and other raw properties in config so we will need the merge.  I will put empty objects as targets to make sure we don't cause any side effects.

x={};require('ut-function.merge')({}, {a:{x}},{a:{x:{y:1}}});x;
// x remains {}

Reply:
https://github.com/softwaregroup-bg/ut-port-http/pull/38/commits/9caf80a58f37ed6092444fac87342845d998548d